proxy/hdrs/Makefile.am

```diff
@@ -87,7 +87,26 @@ test_proxy_hdrs_SOURCES = \
 
 test_proxy_hdrs_LDADD = \
 	$(top_builddir)/src/tscore/libtscore.la \
-    -L. -lhdrs \
+	-L. -lhdrs \
+	$(top_builddir)/src/tscore/libtscore.la \
+	$(top_builddir)/src/tscpp/util/libtscpputil.la \
+	$(top_builddir)/iocore/eventsystem/libinkevent.a \
+	$(top_builddir)/lib/records/librecords_p.a \
+	$(top_builddir)/mgmt/libmgmt_p.la \
+	$(top_builddir)/proxy/shared/libUglyLogStubs.a \
+	@HWLOC_LIBS@ \
+	@LIBCAP@
+
+test_hdr_heap_CPPFLAGS = $(AM_CPPFLAGS) \
+	-I$(abs_top_srcdir)/tests/include
+
+test_hdr_heap_SOURCES = \
+	unit_tests/unit_test_main.cc \
+	unit_tests/test_HdrHeap.cc
+
+test_hdr_heap_LDADD = -L. -lhdrs \
+	$(top_builddir)/src/tscore/libtscore.la \
+	-L. -lhdrs \
```

Comment:
Not an expert about makefile, but is it possible to have some common variable to provide the common LDADD, rather than repeat the whole lists when adding new catch test?

Reply:
Maybe. My experience is that every unit test executable has a unique list. I generally spend some time tweaking it to find all the necessary libraries. That said, PRs welcome!